importer/wdi_importer.py

```diff
@@ -1,63 +1,922 @@
+                        if len(data_values):
+                            if indicator_code in category_vars[category]:
+                                if not global_cat[indicator_code]['saved']:
+                                    newsource = Source(name='World Bank – WDI: ' + global_cat[indicator_code]['name'],
+                                                       description=source_template %
+                                                       (global_cat[indicator_code]['category'],
+                                                        global_cat[indicator_code]['unitofmeasure'],
+                                                        indicator_code,
+                                                        'World Bank – World Development Indicators',
+                                                        global_cat[indicator_code]['source'],
+                                                        '<i style="text-decoration: underline;">Limitations and exceptions: </i><br>' +
+                                                        global_cat[indicator_code]['limitations'] + '<br>' if global_cat[indicator_code]['limitations'] else '',
+                                                        '<i style="text-decoration: underline;">Notes from original source: </i><br>' +
+                                                        global_cat[indicator_code]['sourcenotes'] + '<br>' if global_cat[indicator_code]['sourcenotes'] else '',
+                                                        '<i style="text-decoration: underline;">General comments: </i><br>' + global_cat[indicator_code][
+                                                            'comments'] + '<br>' if global_cat[indicator_code]['comments'] else '',
+                                                        '<i style="text-decoration: underline;">Statistical concept and methodology: </i><br>' +
+                                                        global_cat[indicator_code]['concept'] + '<br>' if global_cat[indicator_code]['concept'] else '',
+                                                        '<i style="text-decoration: underline;">Related source links: </i><br>' +
+                                                        global_cat[indicator_code]['sourcelinks'] + '<br>' if global_cat[indicator_code]['sourcelinks'] else '',
+                                                        '<i style="text-decoration: underline;">Other web links: </i><br>' + global_cat[indicator_code][
+                                                            'weblinks'] + '<br>' if global_cat[indicator_code]['weblinks'] else ''
+                                                        ),
+                                                       datasetId=newdataset.pk)
+                                    newsource.save()
+                                    backup_sql_file.write("INSERT INTO %s (name, description, datasetId, created_at, updated_at) "
```

Comment:
We should never use raw string formatting to construct SQL statements. While obviously we don't expect an injection attack from the World Bank, it is still possible that their data will be unintentionally formatted in a way that breaks the SQL. 

Since we're just generating strings here we can't use parameterized queries... I'd suggest writing a simple string formatting function that just runs any params through pymysql.escape_string first